tests/test_smartbridge.py

```diff
@@ -435,6 +441,138 @@ async def test_device_list(bridge: Bridge):
     assert devices[0]["device_id"] == "3"
 
 
+@pytest.mark.asyncio
+async def test_lip_device_list(bridge: Bridge):
+    """Test methods getting devices."""
+    devices = bridge.target.get_lip_devices()
+    expected_devices = {
```

Comment:
Can we edit down the test data? It seems like one or two would be enough.

Reply:
I'll reduce it to two devices.